src/Services/Kros.ToDos.Api/appsettings.Development.json

```diff
@@ -5,5 +5,10 @@
       "System": "Information",
       "Microsoft": "Information"
     }
+  },
+
+  "ApiJwtAuthorization": {
```

Comment:
Tieto pridane nastavovacky treba dat aj do appsettings.json aby sa mohli nahradzat v release pipeline...

Reply:
Už to tam bolo, ale vymazal som hodnoty a nechal len kľúče.